core/store/src/test/java/uk/gov/gchq/gaffer/store/schema/SchemaTest.java

```diff
@@ -301,10 +301,10 @@ private Schema createSchema() {
                         .description(STRING_TYPE_DESCRIPTION)
                         .build())
                 .type(TestTypes.PROP_MAP, new TypeDefinition.Builder()
-                        .description(MAP_TYPE_DESCRIPTION)
-                        .clazz(LinkedHashMap.class)
-                        .serialiser(mapSerialiser)
-                        .build()
+                                .description(MAP_TYPE_DESCRIPTION)
+                                .clazz(LinkedHashMap.class)
+                                .serialiser(mapSerialiser)
+                                .build()
                 )
```

Comment:
This bracket is causing the formatting change. No need for it to be on its on line.